src/ispypsa/translator/snapshot.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+import pandas as pd
+
+
+def create_snapshot_index(
```

Comment:
See other PR comment about name and comment in temporal_filters.py

Reply:
changed on other PR